src/fetchPerpsGetMarketSummaryWithPriceUpdate.ts

```diff
@@ -0,0 +1,48 @@
+import { ethers } from 'ethers';
+
+export async function fetchPerpsGetMarketSummaryWithPriceUpdate({
+  provider,
+  marketId,
+  PerpsMarketProxyContract,
+  MulticallContract,
+  priceUpdateTxn,
+}: {
+  provider: ethers.providers.BaseProvider;
+  marketId: ethers.BigNumber;
+  PerpsMarketProxyContract: { address: string; abi: string[] };
+  MulticallContract: { address: string; abi: string[] };
+  priceUpdateTxn: {
+    target: string;
+    callData: string;
+    value: number;
+    requireSuccess: boolean;
+  };
+}) {
```

Comment:
```suggestion
}) : Promise<{
  skew: ethers.BigNumber;
  size: ethers.BigNumber;
  maxOpenInterest: ethers.BigNumber;
  currentFundingRate: ethers.BigNumber;
  currentFundingVelocity: ethers.BigNumber;
  indexPrice: ethers.BigNumber;
}> {
```

declare type at exact place we have first encounter the data and avoid custom types for now

Reply:
done